docs/architecture/smart-rollups.mdx

```diff
@@ -1,266 +1,177 @@
-not limited by the bandwidth of layer 1. Thus, the reveal data
-channel can be used to upload large volumes of data to the rollup.
+- A rollup node can request information about the rollup, including the address and origination level of the rollup, known as _metadata requests_.
 
-## Origination
+{/*
+TODO how is this data provided?
+Where does it come from?
+Do we need instructions on how to provide data?
+Eventually include:
+  - importing data from a DAC certificate (which can contain anything ultimately, including a kernel to upgrade to)
+  - revealing data from the (WIP) DAL
+*/}
+
+## Smart Rollup lifecycle
+
+The general flow of a Smart Rollup goes through these phases:
+
+1. Origination: A user called the _rollup operator_ originates the Smart Rollup to layer 1 and one or more users start nodes based on that Smart Rollup to independently verify its operation.
+1. Commitment periods: The Smart Rollup nodes receive the messages in the Smart Rollup inbox, run processing based on those messages, and publish a hash of their state at the end of the period.
+1. Refutation periods: Nodes can publish a concurrent commitment to refute a published commitment.
+1. Triggering outbox messages: When the commitment can no longer be refuted, any client can trigger operations from the Smart Rollup.
+
+Here is more information on each of these phases:
+
+{/* TODO diagram of commitment periods and refutation periods? */}
+
+### Origination
+
+Like smart contracts, users deploy Smart Rollups to layer 1 in a process called _origination_.
+
+The origination process stores data about the rollup on layer 1, including:
+
+- An address for the rollup, which starts with `sr1`
+- The type of proof-generating virtual machine (PVM) for the rollup, which defines the execution engine of the rollup kernel; currently only the `wasm_2_0_0` PVM is supported
+- The code of the rollup, which is a WebAssembly program referred to as the rollup's _kernel_
+- The Michelson data type of the messages it receives from layer 1
+- The genesis commitment that forms the basis for commitments that rollups nodes publish in the future
+
+After it is originated, anyone can run a Smart Rollup node based on this information.
+
+### Commitment periods
+
+Starting from the rollup origination level, levels are partitioned into _commitment periods_ of 60 consecutive layer 1 blocks.
+During each commitment period, each rollup node receives the messages in the rollup inbox, processes them, and updates its state.
+
+Because Smart Rollup nodes behave in a deterministic manner, their states should all be the same if they have processed the same inbox messages with the same kernel starting from the same origination level.
+This state is referred to as the "state of the rollup."
+
+At the end of a commitment period, the next commitment period starts and at the same time a refutation period starts for the commitment period that just ended.
```

Comment:
After discussion with Sylvain, it appears clearly that the refutation does not start at the same time as the next commitment period, but rather when the first commitment for current commitment period is published:
```suggestion
At the end of a commitment period, the next commitment period starts.
```